flask/api/v1/admin/analytics/route.py

```
@@ -0,0 +1,44 @@
+from flask import Blueprint, jsonify, request
```

Ruff (0.11.9)

`flask.request` imported but unused

Remove unused import: `flask.request`

(F401)